app_dart/lib/src/service/luci.dart

```diff
@@ -202,6 +202,7 @@ class LuciService {
     final BatchResponse batchResponse = await buildBucketClient.batch(batchRequest);
     final Iterable<Build> builds = batchResponse.responses
         .map<SearchBuildsResponse>((Response response) => response.searchBuilds)
+        .where((SearchBuildsResponse response) => response.builds != null)
```

Comment:
This will only hide the problem. Do we know why some builds are returning null?

Reply:
This is to open the tree first.
A following up is to find the reason why. Otherwise that empty build will never be updated.

Reply:
LGTM if we will be able to find the issue after we land this change

Reply:
Created https://github.com/flutter/flutter/issues/79393 to track.